rsocket-core/src/test/java/io/rsocket/core/RSocketTest.java

```diff
@@ -556,6 +558,7 @@ protected void init() {
                       .subscribe();
 
                   return Flux.range(1, 10)
+                      .subscribeOn(Schedulers.boundedElastic())
```

Comment:
@OlegDokuka the purpose of these is to decouple the responding since otherwise, with local transports, the responder messages get sent too early before the requester has fully set up the subscriber side.

Reply:
Yeah. That is fine. Actually it has to bee fixed in Reactor, but I postponed that. Thus, once I have it fixed, we can remove it